backend/src/main/java/ioiobagiety/service/impl/ConflictsFinderServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package ioiobagiety.service.impl;
+
+import ioiobagiety.model.classes.Lesson;
+import ioiobagiety.service.ConflictsFinderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ioiobagiety.repository.LessonRepository;
+
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.time.temporal.ChronoUnit;
+import java.time.LocalTime;
+import java.util.Date;
+import java.util.List;
+import java.sql.Time;
+
+@Service
+public class ConflictsFinderServiceImpl implements ConflictsFinderService {
+
+    @Autowired
+    private LessonRepository repository;
+
+    @Transactional
+    public List<Date> getDates() {
+        return repository.getDates();
+    }
+
+    @Transactional
+    public List<Lesson> getSingleDayLessons(Date date) {
+        return repository.getSingleDayLessons(date);
+    }
+
+    @Transactional
+    public List<String> getAllConflicts(){
+        List dates = getDates();
+        List<String> conflicts = new ArrayList<String>();
+        for(Object day: dates){
```

Comment:
czemu tak? czemu nie for(Date day: dates) ?
Wówczas cast poniżej nie będzie potrzebny